code/modules/surgery/organs/tongue.dm

```diff
@@ -310,3 +310,48 @@
 /obj/item/organ/tongue/ethereal/Initialize(mapload)
 	. = ..()
 	languages_possible = languages_possible_ethereal
+
+//Sign Language Tongue - yep, that's how you speak sign language.
```

Comment:
>remove tounge

mute cannot speak

>tie string around tounge

instant fluency in ASL

Reply:
Oh when it says the tongue is tied, it means it in the most literal sense.
No string attached.